app/models/proxy.rb

```diff
@@ -197,9 +197,9 @@ def default_staging_endpoint; end
 
     def default_production_endpoint; end
 
-    def default_staging_endpoint_apiap; end
+    def hosted_default_staging_endpoint; end
```

Comment:
No need to keep this.

IMO, we could now do `hosted_service_deployment_strategy.default_staging_endpoint`.